third_party/terraform/tests/resource_compute_region_per_instance_config_test.go

```diff
@@ -24,7 +24,7 @@ func TestAccComputeRegionPerInstanceConfig_statefulBasic(t *testing.T) {
 		"config_name4":  fmt.Sprintf("instance-%s", randString(t, 10)),
 	}
 	rigmId := fmt.Sprintf("projects/%s/regions/%s/instanceGroupManagers/%s",
-		getTestProjectFromEnv(), "us-central1", rigmName)
+		getTestProjectFromEnv(), getTestRegionFromEnv(), rigmName)
```

Comment:
This change should be equivalent in CI - getTestRegionFromEnv will return us-central1 👍